### ISSUE TITLE
allow npcs to use default weapons

### ISSUE THREAD

**Issue by siroberts27** [Snorps/npcinvasion#2]

**Comment by noideer11** [comment]
Seconding this. Letting NPCs use default weapons was sorely lacking from the previous iterations, and the feature is especially helpful for adding NPCs that use custom weapons.

**Comment by Snorps** [comment]
We'll look into it

**Comment by CombineSlayer24** [comment]
> Seconding this. Letting NPCs use default weapons was sorely lacking from the previous iterations, and the feature is especially helpful for adding NPCs that use custom weapons.

Yes please. I play with a lot of VJ SNPCs, and it was a huge pain in the ass to copy weapon, paste into zinv+, and overall, it made the config a huge mess.
